crates/bevy_ecs/README.md

```diff
@@ -0,0 +1,39 @@
+# Bevy ECS
+
+[![Crates.io](https://img.shields.io/crates/v/bevy_ecs.svg)](https://crates.io/crates/bevy_ecs)
+[![license](https://img.shields.io/badge/license-MIT-blue.svg)](../../LICENSE)
```

Comment:
Will this Link still work when it is shown on crates.io?

Reply:
I wondered the same and took a look at how crates.io renders the license link in the root readme. `./LICENSE` is correctly translated into the full link to the repo, so I hope the same will work here.

Reply:
To be safe I replaced the relative link with a full link to the bevy repo.